pkg/ctl/cmdutils/cmdutils.go

```diff
@@ -173,6 +173,11 @@ func AddUpdateAuthConfigMap(fs *pflag.FlagSet, updateAuthConfigMap *bool, descri
 	fs.BoolVar(updateAuthConfigMap, "update-auth-configmap", true, description)
 }
 
+// AddClusterEndpointOverrideFlag adds common cluster-endpoint-override flag
+func AddClusterEndpointOverrideFlag(fs *pflag.FlagSet, clusterEndpoint *string) {
+	fs.StringVar(clusterEndpoint, "cluster-endpoint-override", "", "cluster endpoint to use instead of the eks cluster endpoint provided by Cloudformation")
```

Comment:
it feels like we wouldn't need the `-override` suffix since cli flags generally take precedence anyway. i would just use `--cluster-endpoint` here.

Reply:
Yeah, very good point.